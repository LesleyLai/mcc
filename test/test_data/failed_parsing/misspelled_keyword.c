int main(void) {
    returns 0;
}