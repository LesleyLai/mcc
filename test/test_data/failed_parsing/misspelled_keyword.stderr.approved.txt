{{base_dir}}/failed_parsing/misspelled_keyword.c:2:13: Error: expect ';'
2 |         returns 0;
  |                 ^

{{base_dir}}/failed_parsing/misspelled_keyword.c:2:14: Error: Expect end of the file
2 |         returns 0;
  |                  ^

